bumble/host.py

```diff
@@ -484,6 +487,10 @@ def on_hci_acl_data_packet(self, packet):
         if connection := self.connections.get(packet.connection_handle):
             connection.on_hci_acl_data_packet(packet)
 
+    def on_hci_sco_data_packet(self, packet):
+        # Look for the connection to which this data belongs
```

Comment:
I don't think this comment applies here.

Reply:
Done